client/shaders/second_stage/opengl_fragment.glsl

```diff
@@ -58,6 +59,12 @@ vec4 applyToneMapping(vec4 color)
 	color.rgb *= whiteScale;
 	return color;
 }
+
+vec3 applySaturation(vec3 color, float factor)
+{
+       float brightness = dot(color, vec3(0.2125, 0.7154, 0.0721));
```

Comment:
black magic. what do these numbers mean?

Reply:
See original PR and author.

Reply:
is that a power scale of RGB w.r.t the influence on human eyeball photo-receptors?

Reply:
seems i've stepped into a discussion i don't understand, but i still suggest explaining random numbers w/ a comment.

Reply:
What this does is creating a luminosity of a color given in RGB, taking into account that the human eye is most sensitive to green and least to blue.

I just trusted the original author for the specific values. But you raised a good question, so I googled around for it, see: https://www.w3.org/WAI/GL/wiki/Relative_luminance

Reply:
There should be a comment for this

Reply:
Done.